jhub_apps/service/routes.py

```diff
@@ -215,14 +251,64 @@ async def update_server(
 @router.delete("/server/{server_name}")
 @router.delete("/server/")
 async def delete_server(
-    user: User = Depends(get_current_user),
-    server_name=None,
+    server_name: str = None,
     remove: bool = False,
+    user: User = Depends(get_current_user),
 ):
```

Comment:
no changes should be made for delete server, we don't plan to let the user delete shared server (not owned by the user)